consumer.go

```diff
@@ -66,7 +66,17 @@ func (c *Consumer) Run(ctx context.Context) {
 	c.log.Printf("Started working...")
 	<-ctx.Done()
 
-	err := c.consumer.Close()
+	// need to unassign to be sure the following Close()
+	// will return
+	// https://github.com/confluentinc/confluent-kafka-go/issues/189
```

Comment:
Better add an explicit TODO so that we're reminded to get rid of this when the issue is fixed.